environments/production/proxy.yml

```diff
@@ -23,3 +23,35 @@ special_sites:
   - wiki_http
   - motech
   - motech2
+
+limit_req_zones:
+  - name: "restore"
+    zone: "$digest_submission"
+    size: "10m"
+    rate: "1r/s"
+  - name: "submissions"
+    zone: "$digest_submission"
+    size: "10m"
+    rate: "10r/s"
+
+site_locations:
+  - name: "/a/sauti-1/phone/restore"
```

Comment:
any reason not to use a regex and match all domains?

Reply:
This PR is for intentionally rate-limiting just the one domain. (There's still a separate thing I want to do which is to send restores to the new split-out "mobile_webworkers".)